### ISSUE TITLE
options.multiple appears to incorrectly set fileLimit to 1

### ISSUE THREAD

**Issue by keithj2780** [chanced/filedrop-svelte#1]
I'd expect leaving options.multiple as the default & setting options.fileLimit to anything > 1 to allow multiple files to be uploaded.

Is the a ! missing from this code ?
```
export function processFiles(files: FileWithPath[], options: FileDropOptions): Files {
	let { fileLimit } = options;
	if (options.multiple) {
		fileLimit = 1;
	}
```

If not, then what options do I set to upload multiple files ?

**Comment by chanced** [comment]
Ah, yea. I definitely missed a ! there. My apologizes.

**Comment by chanced** [comment]
That should be fixed. I also added harnesses to properly test this. Hopefully I'll have decent coverage early next week.

I spot checked the fix and pushed a new version (0.0.10). 

```typescript
if (options.multiple != undefined && !options.multiple) {
	fileLimit = 1;
}
```

**Comment by keithj2780** [comment]
Many thanks for the fast fix.  0.0.10 is working for me now :)

Cheers Keith
